src/actionlist/action-list-item.scss

```diff
@@ -22,6 +22,7 @@
   list-style: none;
   background-color: transparent;
   border-radius: $border-radius;
+  outline: $border-style $border-width var(--color-action-list-item-default-hover-border);
```

Comment:
Using `outline` to not disrupt our very particular item sizing with a `border`. Is there any reason _not_ to do this? 🤔

Reply:
I think that's fine as long as we're ok with Safari not having rounded corners? Hopefully just a matter of time. `box-shadow` might be an alternative, but I think that won't be seen in "Windows high contrast mode".

Reply:
Also scoped to active/hover state